enterprise/server/cmd/goinit/main.go

```diff
@@ -323,6 +323,16 @@ func main() {
 	}
 	die(os.WriteFile("/etc/hosts", []byte(strings.Join(hosts, "\n")), 0755))
 	die(os.WriteFile("/etc/resolv.conf", []byte("nameserver 8.8.8.8"), 0755))
+	if _, err := os.Stat("/etc/mtab"); err != nil {
+		if os.IsNotExist(err) {
+			die(syscall.Symlink("/proc/mounts", "/etc/mtab"))
+			log.Infof("Wrote /etc/mtab")
+		} else {
+			die(err)
+		}
+	} else {
+		log.Infof("Found existing /etc/mtab")
+	}
```

Comment:
do we need the log.Infofs here? I'm not even sure where these logs show up -- the console?

Reply:
They show up when running firecracker in debug mode. Got rid of them though since they aren't super useful